src/components/ProjectSettingsTabs/types.ts

```diff
@@ -0,0 +1,16 @@
+import { ProjectSettingsQuery } from "gql/generated/types";
+
+export type GeneralTabProps = {
+  data: Pick<
+    ProjectSettingsQuery["projectSettings"]["projectRef"],
+    | "enabled"
```

Comment:
I'm not sure why line 6 has a `|` prepended. VSC seems to supply it automatically after 6 keys are provided.